CONTRIBUTING.md

```diff
@@ -229,6 +229,25 @@ Run script to download dashboards to `./dashboards` folder
 node scripts/download_dashboards.mjs
 ```
 
+## Contributing to Documentation
+
+When submitting PRs for documentation updates, build the docs locally and ensure functionality before submission. To build docs locally, ensure you have Python installed, then follow these steps:
+
+```sh
+pip install -r docs/requirements.txt
+yarn build:docs
+cd docs
+mkdocs serve --watch pages
+```
```

Comment:
Could be simplified if we decide to add a yarn script for each command, also would prefer to not switch directories
- https://github.com/ChainSafe/lodestar/pull/6296